boards/px4/fmu-v5x/src/manifest.c

```diff
@@ -163,6 +184,9 @@ static px4_hw_mft_list_entry_t mft_lists[] = {
 	{V5X01, hw_mft_list_v0500, arraySize(hw_mft_list_v0500)}, // FMUV5X,                 Rev 1
 	{V5X02, hw_mft_list_v0500, arraySize(hw_mft_list_v0500)}, // FMUV5X,                 Rev 2
 	{V5X10, hw_mft_list_v0510, arraySize(hw_mft_list_v0510)}, // NO PX4IO,               Rev 0
+	{V5X40, hw_mft_list_v0540, arraySize(hw_mft_list_v0540)}, // FMUV5X,  HB CM4 base    Rev 0
+	{V5X41, hw_mft_list_v0540, arraySize(hw_mft_list_v0540)}, // FMUV5X,  HB CM4 base    Rev 1
+	{V5X42, hw_mft_list_v0540, arraySize(hw_mft_list_v0540)}, // FMUV5X,  HB CM4 base    Rev 2
```

Comment:
Did all 3 revisions actually ship?

Reply:
Rev 1 and Rev 2, yes. And Rev 0 is supposedly Skynode.